src/appleseed.studio/mainwindow/rendering/renderingmanager.h

```diff
@@ -162,15 +161,16 @@ class RenderingManager
 
   private:
     StatusBar&                                  m_status_bar;
-    QtRendererController                        m_renderer_controller;
+    QtRendererController                        m_renderer_controller; 
 
     renderer::Project*                          m_project;
     renderer::ParamArray                        m_params;
     foundation::SearchPaths                     m_resource_search_paths;
     RenderingMode                               m_rendering_mode;
     RenderTab*                                  m_render_tab;
 
-    std::unique_ptr<QtTileCallbackFactory>      m_tile_callback_factory;
+    std::unique_ptr<renderer::TileCallbackCollectionFactory>      
```

Comment:
You need to include the proper header.

Reply:
If you are talking about header for `TileCallbackCollectionFactory`, I have already included it in `src/appleseed/renderer/api/rendering.h` and then added `rendering.h` in the file.

Do you still want me to include the header for `TileCallbackCollectionFactory` ?

And `#include <memory>` is already there for `std::unique_ptr`.

Reply:
Oh sorry, my bad. In that case no we don't want to include `tilecallbackcollection.h` directly. And indeed <memory> is already included. Apologies!